From 7b549334d35c5fbb5feaaba2c63d0484696db283 Mon Sep 17 00:00:00 2001
From: Jiangli Yuan <jlyuan@motorola.com>
Date: Tue, 26 Jan 2016 13:44:17 -0800
Subject: [PATCH 648/959] IKSWM-16884:8916/defconfig: Enable Arm/Neon/AES
 kernel options

 Enable rm/Neon/AES kernel options

  CONFIG_CRYPTO_SHA256_ARM
  CONFIG_KERNEL_MODE_NEON
  CONFIG_CRYPTO_AES_ARM_BS

Change-Id: I5b50f6d0253d0dbe323fa13d02eea2615c8b901f
Signed-off-by: Jiangli Yuan <jlyuan@motorola.com>
Reviewed-on: http://gerrit.mot.com/816290
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Zhi-Ming Yuan <a14194@motorola.com>
Reviewed-by: Mark Reed <mreed@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 0dba08f337d..02e99343244 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -61,3 +61,6 @@ CONFIG_ALSA_TO_H2W=y
 CONFIG_ESD_FS=y
 CONFIG_SEC_DRIVER=y
 CONFIG_MMC_CQ_HCI=y
+CONFIG_CRYPTO_SHA256_ARM=y
+CONFIG_KERNEL_MODE_NEON=y
+CONFIG_CRYPTO_AES_ARM_BS=y
-- 
2.11.0

